lib/pam/pam.go

```diff
@@ -35,7 +35,11 @@ package pam
 // extern int _pam_acct_mgmt(void *, pam_handle_t *, int);
 // extern int _pam_open_session(void *, pam_handle_t *, int);
 // extern int _pam_close_session(void *, pam_handle_t *, int);
+// extern char **_pam_getenvlist(void *handle, pam_handle_t *pamh);
```

Comment:
No spacing between `**` and the function name - presumably this is all flattened out by the compiler anyway because function names can't contain `**`?

Reply:
We don't really have a C style guide that we follow as far as I know. That being said, but I have been trying to follow how it's declared in the PAM man pages.

```
NAME
       pam_getenvlist - getting the PAM environment

SYNOPSIS
       #include <security/pam_appl.h>

       char **pam_getenvlist(pam_handle_t *pamh);
```